src/formatter_md.py

```diff
@@ -113,3 +113,39 @@ def add_horizontal(self):
                 horizontal(str): horizontal rule string
         """
         return '---'
+
+    def _prepare_headers_cells(self, headers):
+        """Prepare headers cells for table creation.
+
+        Args:
+            headers (list): table column names
+
+        Returns:
+            headers_cells(list): headers cells ready for further table-related
+            operations
+        """
+        headers_row = ['{header} | '.format(header=h) for h in headers]
```

Comment:
can be shortened to: 
headers_cells = [' | '.join(headers), '--- | ' * len(headers)]
So it can be placed inside create_table(), not in a separated function

Reply:
Sure, but do you want to have possibility to reuse this function in future with different kinds of tables?

Reply:
other tables have different style of headers, so this function have to be overloaded anyway. In the future, in overloaded class we will write a helper function like that, if it will be needed.

Reply:
done